src/apply_imputation_link.py

```diff
@@ -55,34 +55,30 @@ def create_and_merge_imputation_values(
     imputation_config = {
         "c": {
             "intermediate_column": "constructed",
-            "marker": "C",
+            "marker": "c",
             # doesn't actually apply a fill so can be forward or back
             "fill_column": auxiliary,
             "fill_method": "ffill",
             "link_column": construction_link,
         },
         "fir": {
             "intermediate_column": "fir",
-            "marker": "FIR",
+            "marker": "fir",
             "fill_column": target,
             "fill_method": "ffill",
             "link_column": cumulative_forward_link,
         },
         "bir": {
             "intermediate_column": "bir",
-            "marker": "BIR",
+            "marker": "bir",
             "fill_column": target,
             "fill_method": "bfill",
             "link_column": cumulative_backward_link,
         },
         "fic": {
-            # FIC only works if the C is in the first period of the business being
```

Comment:
Has this issue been resolved?

Reply:
@robertswh i think it is resolved?

Reply:
By accident i removed these comments from the code, i will re add them in the original code. 

This pull request is not related what is being mentioned in the comments, the comments are related to the manual construction methods and things we must be aware of.